src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -148,6 +149,23 @@ public PeriodicBackupRunner(DocumentDatabase database, ServerStore serverStore)
             };
         }
 
+        private Dictionary<string, long> UpdateTombstoneLastEtagsPerCollection(DocumentsOperationContext context, Dictionary<string, long> existingData)
+        {
+            existingData = existingData ?? new Dictionary<string, long>(); //precaution, just in case
```

Comment:
No, this will hide errors down the line, better to fail with NRE. 
this is only called internally.